tests/ext/sandbox/exceptions_are_passed_to_the_tracing_closure.phpt

```diff
@@ -0,0 +1,56 @@
+--TEST--
+Exceptions from original call are passed to tracing closure (PHP 7)
+--SKIPIF--
+<?php if (PHP_VERSION_ID < 70000) die('skip PHP 5 tested in separate test'); ?>
+--FILE--
+<?php
+use DDTrace\SpanData;
+
+register_shutdown_function(function () {
+    array_map(function($span) {
+        echo $span['name'];
+        if (isset($span['meta']['error.msg'])) {
+            echo ' with exception: ' . $span['meta']['error.msg'];
+        }
+        echo PHP_EOL;
+    }, dd_trace_serialize_closed_spans());
+});
+
+function testExceptionIsNull()
+{
+    echo "testExceptionIsNull()\n";
+}
+
+function testExceptionIsPassed()
+{
+    echo "testExceptionIsPassed()\n";
+    throw new Exception('Oops!');
+}
+
+dd_trace_function('testExceptionIsNull', function (SpanData $span, array $args, $retval, $ex) {
+    $span->name = 'TestNull';
+    var_dump($ex === null);
+});
+
+dd_trace_function('testExceptionIsPassed', function (SpanData $span, array $args, $retval, $ex) {
+    $span->name = 'TestEx';
+    var_dump($ex instanceof Exception);
+});
```

Comment:
Would it make sense to add test that throws an exception inside dd_trace_function. At the same time when exception is passed to the hook function, then asserting that the outer exception is the only one that gets "uncaught"?

Also I wonder about scenario like this:

```
class SubException extends Exception { }

function a(){
  throw new SubException('Oops!');
}

dd_trace_function('a', function( $s, $args, $r, $ex) {
    throw new Exception("!");
});

try {
  a();
catch (SubException $ex) {
   echo "Caught SubException" ;
}
```
With no uncaught exception reported.

Also now just thinking what would happen when someone evil or by mistake does something like this:
```
dd_trace_function('a', function( $s, $args, $r, $ex) {
    throw $ex;
});
```
Its evil, but would be nice to catch this error in other ways than crash and burn :)